web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -161,6 +166,39 @@ describe("<BusinessFormationPaginator />", () => {
       await page.stepperClickToBillingStep();
       expect(screen.getByTestId("help-button")).toBeInTheDocument();
     });
+
+    it("the help button fires analytics on every page", async () => {
```

Comment:
pebble - in other tests we've typically separated each of the assertions into their own test. You can find this example in the onboarding test files. There are instances where we do integration tests, I believe there are large tests in formation. But I think smaller unit test that test one thing is preferred.

it fires on tab 1
it fires on tab 2, etc

Reply:
Yeah, thats fine. Historically I've not had super hard preferences but in hind sight granularity is probably not a bad thing. 

I will fix